numba/cuda/ufuncs.py

```diff
@@ -22,6 +22,7 @@ def get_ufunc_info(ufunc_key):
 def ufunc_db():
     # Imports here are at function scope to avoid circular imports
     from numba.cpython import cmathimpl, mathimpl, numbers
+    from numba.np import npyfuncs
     from numba.np.npyfuncs import _check_arity_and_homogeneity
     from numba.np.npyfuncs import (np_complex_acosh_impl, np_complex_cos_impl,
                                    np_complex_sin_impl)
```

Comment:
Just a very minor suggestion: now that we're importing all of `npyfuncs` it would be a nice little tidy-up to remove these imports and prefix references to these functions with `npyufuncs.`